bin/cli.js

```diff
@@ -65,18 +63,13 @@ const cliFlags = yargs
     return true;
   }).argv;
 
-//Get all explicitly terminal set options, does not include url because url has no default option
-const flags = Object.keys(yargs.getOptions().default).reduce((accum, prop) => {
-  if (cliFlags[prop] !== yargs.getOptions().default[prop])
-    accum[prop] = cliFlags[prop];
-  else if (cliFlags.config && cliFlags.config.flags && cliFlags.config.flags[prop])
-    accum[prop] = cliFlags.config.flags[prop];
-  else accum[prop] = yargs.getOptions().default[prop];
-  return accum;
-}, {});
 
-//Merge options from all sources. Order indicates precedence (last => most important)
-let options = Object.assign({}, cliFlags.config, { flags: flags });
+const config = cliFlags.config ? getConfigFromFile(cliFlags.config) : {};
+
+// For the config flags and URL, we'll allow CLI to override what's set in the config Object
```

Comment:
IMO if a user explicitly sets a flag in the CLI I think he/she wants to trump the config file. Otherwise they'll have to remember to check the config file. Thoughts @denar90 ?